packages/repack/commands.js

```diff
@@ -1,43 +1,33 @@
 const path = require('path');
-const { createRequire } = require('module');
 
 function getReactNativeCliPath() {
   let cliPath;
 
-  try {
-    cliPath = path.dirname(require.resolve('@react-native-community/cli'));
-  } catch {
-    // NOOP
-  }
-
   try {
     cliPath = path.dirname(
-      require.resolve('react-native/node_modules/@react-native-community/cli')
+      require.resolve('@react-native/community-cli-plugin')
```

Comment:
This will break Re.Pack working with 0.71 or 0.72, isn't it? Couldn't we look for `"bin"` entry in `react-native` package, which points to the cli?

Reply:
> Couldn't we look for "bin" entry in react-native package, which points to the cli?

I don't think so ;/ Of course we can get path to the `cli.js` and it will be resolved, but how can we get informations about commands from there? 🤔

Reply:
Unfortunately we can't go with the easy path, because right now `start` and `bundle` command are added in `react-native.config.js` so they won't exist under `/commands` dir.